### ISSUE TITLE
Move Schwarmstedt from Umland to own domain

### ISSUE THREAD

**Issue by CodeFetch** [freifunkh/site#57]
With more than 300 nodes Umland is already twice as big as our second biggest domain. Schwarmstedt is the biggest mesh in Umland and should therefore be moved to a dedicated domain.

**Comment by CodeFetch** [comment]
Hm... When thinking about it, this might confuse users. Instead I propose to add a prefix to the names of the existing domains which reflect districts of Hannover e.g. "Hannover: Misburg" and then split the Umland domain further into regions of the region of Hannover like in this map with the prefix like e.g. "Region: Neustadt", "Region: Springe": https://www.hannover.de/Leben-in-der-Region-Hannover/Gesundheit/Gesundheitsschutz/Coronavirus-in-der-Region-Hannover/Infektionen-mit-COVID-19-in-der-Region-Hannover

**Comment by 1977er** [comment]
Do we have/need code to move nodes from ie. `Sehnde` to `Umland: Sehnde`?

**Comment by Dark4MD** [comment]
Sounds good.
For the existing Domains it would just be a display name change in the site and the meshviewer, if we leave the internal names without the prefixes like it is right now.

**Comment by Dark4MD** [comment]
Would look something like this (i hope i got all Region and Umland definitions right)
https://github.com/Dark4MD/site/commit/b5b12589f427296a7e5f514261f8d521b86078ad
Btw. we still have legacy as default domain. I'm not sure which would be best but maybe we do something like "Other" or "Umland: Other" as default.

**Comment by 1977er** [comment]
@Dark4MD The domX files are generated from `gen.py`, aren't they?

**Comment by CodeFetch** [comment]
@1977er Yes, using gen.py. Manu is right. Only the Umland nodes will be moved to new domains and the rest are just name changes.

**Comment by AiyionPrime** [comment]
> Btw. we still have legacy as default domain. I'm not sure which would be best but maybe we do something like "Other" or "Umland: Other" as default.

I don't think we should put them in an already big domain like umland, but implement a default domain like leetfeld? for this purpose.

Maybe we could even setup some sort of notification on the nodes cli, when the domain is not set properly.

**Comment by bschelm** [comment]
Please remember to add the domain-switcher to the next firmware, to push the routers starting with sgs* to the domain Schwarmstedt. There are still too many routers in Schwarmstedt in the domain "Umland".

**Comment by AiyionPrime** [comment]
This requires Firmware newer than vH25.

**Comment by AiyionPrime** [comment]
...and is therefore effectively blocked by #87
